blackbox/docs/azure_discovery.txt

```diff
@@ -0,0 +1,77 @@
+.. highlight:: yaml
+.. _azure_discovery:
+
+===============
+Azure Discovery
+===============
+
+When running Crate on Microsoft Azure you usually face
+the problem that Crate's default discovery mechanism does
+not work out of the box.
+
+Luckily Crate has a built-in mechanism for unicast host
+discovery in an Azure environment.
+
+Requirements to use Azure api for node discovery
+------------------------------------------------
+
+The mechanisms for discovery on Microsoft Azure are built
+in with Crate, but the required sdk libraries are not included in
+any Crate distribution and needed to be added to Crate's
+libs folder. By default this is
+``$CRATE_HOME/lib``
```

Comment:
no don't do this. we have a plugin folder for this, so the right way is to install them in `plugins/cloud-azure/libs` folder. never move plugin related libs into the global lib path, if some want to remove azure support, you must now all related libs otherwise which is kind a wrong... thats why we support loading libs from `plugins/<plugin</libs`